public/js/post/course_components.js

```diff
@@ -34,14 +34,14 @@ var CourseCode = React.createClass({
             // special case for calculus requirement since it doesn't fit the same pattern
             return (
                 <p className='code'>
-                    (<span id='mat135' className='courseName' onClick={this.openModal}>MAT135H</span> and
-                    <span id='mat136' className='courseName' onClick={this.openModal}>MAT136H</span>) or
+                    <span id='mat135' className='courseName' onClick={this.openModal}>( MAT135H</span> and
```

Comment:
This change isn't great (it makes the parenthesis part of the styling). Instead, I recommend removing the padding from `courseName` elements. You might need to add some new padding in other places, and fix some other text, but that's a better change to make.